bentoml/adapters/__init__.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from bentoml.adapters.base_input import BaseInputAdapter
+from bentoml.adapters.dataframe_handler import DataframeHandler
+from bentoml.adapters.tensorflow_tensor_handler import TensorflowTensorHandler
+from bentoml.adapters.json_handler import JsonHandler
+from bentoml.adapters.image_handler import ImageHandler
+from bentoml.adapters.legacy_image_handler import LegacyImageHandler
+from bentoml.adapters.fastai_image_handler import FastaiImageHandler
+from bentoml.adapters.clipper_handler import (
+    ClipperBytesHandler,
+    ClipperDoublesHandler,
+    ClipperFloatsHandler,
+    ClipperIntsHandler,
+    ClipperStringsHandler,
+)
+
+from bentoml.adapters.dataframe_output import DataframeOutput
+from bentoml.adapters.tf_tensor_output import TfTensorOutput
+from bentoml.adapters.base_output import BaseOutputAdapter
+from bentoml.adapters.default_output import DefaultOutput
+from bentoml.adapters.json_output import JsonserializableOutput
+
+
+DataframeInput = DataframeHandler
+TfTensorInput = TensorflowTensorHandler
+JsonInput = JsonHandler
+ImageInput = ImageHandler
+LegacyImageInput = LegacyImageHandler
+FastaiImageInput = FastaiImageHandler
+
+ClipperBytesInput = ClipperBytesHandler
```

Comment:
Maybe make it the other way around?

E.g. rename ImageHandler class to ImageInput, and add `ImageHandler = ImageInput` here?

It will be easier to deprecate those alias later on, and easier to new contributors to understand the code base.

Let me know if you plan to do that in a separate PR

Reply:
Yeah, working in progress.

Reply:
In this PR

Reply:
Actually, would be nice to add a deprecation warning when the user is using the "handler" APIs. Would something like this work?

```
class ImageHandler(ImageInput):
   """ backwards compatible alias ..... """

  def __init__(...):
         super...
         logger.warning("will be deprecated .... use ImageInput")
 
```